modules/tinymce/src/core/test/ts/browser/EditorRemoveTest.ts

```diff
@@ -88,4 +88,18 @@ describe('browser.tinymce.core.EditorRemoveTest', () => {
     const editor = await McEditor.pFromHtml<Editor>('<textarea id="tinymce" style="display: block;"></textarea>', settings);
     testRemoveStyles(editor, 'block');
   });
+
+  it('remove editor that unbinds mousedown in the remove handler', async () => {
+    const editor = await McEditor.pFromHtml('<textarea></textarea>', {
```

Comment:
```suggestion
    const editor = await McEditor.pFromSettings({
```
This just uses the default HTML so we can just shorten this a little.

Reply:
ah nice I was just following the style of the test I copied this from 😂